README.md

```diff
@@ -1,2 +1,57 @@
 # strike-off-objections-api
+
+### Overview
 API for handling objections to a company being struck off the register
+
+### Requirements
+
+In order to run the service locally you will need the following:
+
+- [Java 8](http://www.oracle.com/technetwork/java/javase/downloads/jdk8-downloads-2133151.html)
+- [Maven](https://maven.apache.org/download.cgi)
+- [Git](https://git-scm.com/downloads)
+- [MongoDB](https://www.mongodb.com/)
+
+### Getting started
+
+To checkout and build the service:
+1. Clone [Docker CHS Development](https://github.com/companieshouse/docker-chs-development) and follow the steps in the README.
+2. Run ./bin/chs-dev modules enable strike-off-objections
+3. Run ./bin/chs-dev development enable strike-off-objections-api (this will allow you to make changes).
+4. Run docker using "tilt up" in the docker-chs-development directory.
+5. Use spacebar in the command line to open tilt window - wait for strike-off-objections-web to become green.
+6. Open your browser and go to page http://chs.local/strike-off-objections
```

Comment:
As this is the api probably better to have a list of endpoints in the readme, and direct them to hit these. As this url is for the strike-off-api

Reply:
As there is now a table of endpoints, i'd change step 6 to something like "Send a rest request to one of the api's endpoints"